### PR TITLE
DOCSP-37703 Document how to create indexes with options in Atlas CLI

### PR DESCRIPTION
**Pull request by kanchana-mongodb**
- [DOCSP-37703](https://jira.mongodb.org/browse/DOCSP-37703)
- STAGING
  - https://preview-mongodbkanchanamongodb.gatsbyjs.io/atlas-cli/DOCSP-37703/reference/json/search-index-config-file/
  - https://preview-mongodbkanchanamongodb.gatsbyjs.io/atlas-cli/DOCSP-37703/reference/json/rolling-index-config-file/
- [LATEST BUILD LOG](https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=660acd6f3a5920a291ebd48e) (contains errors not introduced by changes in this PR)

### Self-Review Checklist

- [ ] Check that the submodule pulled in the right changes (if applicable).
- [ ] [Define](https://wiki.corp.mongodb.com/display/DE/Taxonomy+tagging+instructions) taxonomy [values](https://wiki.corp.mongodb.com/display/DE/Docs+Taxonomy) at top of page.
- [ ] Add genre facets (tutorial or reference), as in this [example PR](https://github.com/10gen/cloud-docs/pull/5042).
- [ ] Add programmingLanguage (if necessary).
- [ ] Add meta keywords (if necessary).
- [ ] Resolve any new warnings or errors in the build.
- [ ] Proofread for spelling and grammatical errors.
- [ ] Check staging for rendering issues.
- [ ] Confirm links are working.

### CODE REVIEW

File: source/reference/json/search-index-config-file.txt [hunk 1]
```diff
@@ -10,112 +10,55 @@ Atlas Search Index Configuration File
...
 
-You can use an |fts| index configuration file to specify the required 
-settings for creating a search index using the {+atlas-cli+}. The 
-{+atlas-cli+} accepts ``.json`` search index configuration files. You
-can use the  ``.json`` search index configuration file to create the
-index for :ref:`cloud <atlas-clusters-search-indexes-create>` and
-:ref:`local <atlas-deployments-search-indexes-create>` Atlas
-deployments. 
+.. |atlas-cli command| replace:: :ref:`atlas-clusters-search-indexes-create`
+.. |configuration-file-name| replace:: atlas search indexes configuration
```
Comment (sarahsimpers) [inline source/reference/json/search-index-config-file.txt:14]:
```suggestion
.. |configuration-file-name| replace:: search index configuration  
```

File: source/reference/json/search-index-config-file.txt [hunk 2]
```diff
@@ -10,112 +10,55 @@ Atlas Search Index Configuration File
...
    :class: singlecol
 
-You can use an |fts| index configuration file to specify the required 
-settings for creating a search index using the {+atlas-cli+}. The 
-{+atlas-cli+} accepts ``.json`` search index configuration files. You
-can use the  ``.json`` search index configuration file to create the
-index for :ref:`cloud <atlas-clusters-search-indexes-create>` and
-:ref:`local <atlas-deployments-search-indexes-create>` Atlas
-deployments. 
+.. |atlas-cli command| replace:: :ref:`atlas-clusters-search-indexes-create`
```
Comment (sarahsimpers) [inline source/reference/json/search-index-config-file.txt:13]: Can we add the facet info on this page? 

```
.. facet::
   :name: genre
   :values: tutorial
```

**Review by sarahsimpers — APPROVED** [review]
LGTM with two minor comments. Thanks for doing this work!

**Review by gssbzn — COMMENTED** [review]
The associated ticket relates to https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v2/#tag/Rolling-Index/operation/createRollingIndex which is the same options as https://www.mongodb.com/docs/manual/core/indexes/create-index/ and unrelated to search indexes

File: source/reference/json.txt [hunk 3]
```diff
@@ -150,6 +154,17 @@ To learn more about alert configuration files, see
...
 
+Rolling Index Configuration File 
+--------------------------------
+
+To learn more about rolling index configuration files, see 
+:ref:`atlas-cli-rolling-index-config-file`. For example files, see:
+
+- `Unique indexes <https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/e2e/atlas/data/create_2dspere_index.json>`__ 
+- `Sparse Index <https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/e2e/atlas/data/create_sparse_index.json>`__ 
+- `Partial indexes <https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/e2e/atlas/data/create_partial_index.json>`__
```
Comment (sarahsimpers) [inline source/reference/json.txt:165]: Can we add these links as a "See Also" to the dedicated page as well so they don't get missed? Thank you!

**Review by sarahsimpers — APPROVED** [review]
LGTM with one suggestion

**Review by gssbzn — APPROVED** [review]
LGTM

**Pull request merged**